server.js

```diff
@@ -1659,8 +1654,7 @@ app.get("/data/elograph.json", asyncMiddleware(async(req, res) => {
                 elo = CalculateEloFromPercent(fakewins / fakecount);
             }
 
-            if (match.network1 == "d13c40993740cb77d85c838b82c08cc9c3f0fbc7d8c3761366e5d59e8f371cbd"
-                && match.network2 == "62b5417b64c46976795d10a6741801f15f857e5029681a42d02c9852097df4b9") {
+            if (match.network1 == ELF_NETWORKS[1] && match.network2 == ELF_NETWORKS[0]) {
```

Comment:
To be future compatible we could check ELF_NETWORKS.includes for both network 1 and network 2.

Reply:
Added includes check for both and added comment too.